src/tests/vos_perf.c

```diff
@@ -887,6 +893,12 @@ main(int argc, char **argv)
 
 	ts_update_or_fetch_fn = vos_update_or_fetch;
 
+#ifdef ULT_MMAP_STACK
+	rc = stack_pool_create(&sp);
```

Comment:
similar to prior comment, if stack_pool_create is defined as 0 when ULT_MMAP_STACK is undefined, it will just work as is.   Compiler is smart enough to remove the unnecessary check.

Reply:
In header you would put something like this:

#ifdef ULT_MMAP_STACK
int
stack_pool_create(...);
#else 
#define stack_pool_create(...) 0
#endif  

In C files you would just have this without the inline ifdef

rc = stack_pool_create(&sp);
if (rc)
    return -1;